seeds/facility-fake-data.js

```diff
@@ -3,54 +3,291 @@
+				const facilityServices = FacilityService.create(
+					facilityServiceData
+				);
+				const facilityUtilityData = {
+					facility_id: values[0].id,
+					utility_id: utilityId,
+				};
+				const facilityUtilities = FacilityUtility.create(
+					facilityUtilityData
+				);
+				const geolocationData = {
+					datum: datam,
+					geolocation: geopointData,
+					facility_id: values[0].id,
+				};
+				const geolocations = Geolocation.create(geolocationData);
+				const locationData = {
+					catchment_area: catchmentArea,
+					catchment_population: catchmentPopulation,
+					district_id: districtId,
+					facility_id: values[0].id,
+				};
+				const locations = Location.create(locationData);
+
+				Promise.all([
+					facilityResources,
+					addresses,
+					contacts,
+					facilityServices,
+					facilityUtilities,
+					geolocations,
+					locations,
+				]).then(resvalues => {
+					console.log(
+						"All dependencies in association with\
+					 facilities are created"
+					);
+				});
 			});
 		} else {
 			Promise.all([facilityFake]).then(values => {
-				console.log(values[0]);
+				const facilityResourceData = {
+					facility_id: values[0].id,
+					resource_id: resourceId,
+					quantity: resourceQuantity,
+					description: faker.lorem.sentence(),
+				};
+				const facilityResources = FacilityResource.create(
+					facilityResourceData
+				);
+				const addressData = {
+					physical_address: physicalAddress,
+					postal_address: postalAddress,
+					facility_id: values[0].id,
+				};
+				const addresses = Address.create(addressData);
+				const contactData = {
+					contact_person_fullname: fullname,
+					contact_person_phone: phone,
+					contact_person_email: email,
+					facility_id: values[0].id,
+				};
+				const contacts = ContactPeople.create(contactData);
+				const facilityServiceData = {
+					service_id: serviceId,
+					facility_id: values[0].id,
+				};
+				const facilityServices = FacilityService.create(
+					facilityServiceData
+				);
+				const facilityUtilityData = {
+					facility_id: values[0].id,
+					utility_id: utilityId,
+				};
+				const facilityUtilities = FacilityUtility.create(
+					facilityUtilityData
+				);
+				const geolocationData = {
+					datum: datam,
+					geolocation: geopointData,
+					facility_id: values[0].id,
+				};
+				const geolocations = Geolocation.create(geolocationData);
+				const locationData = {
+					catchment_area: catchmentArea,
+					catchment_population: catchmentPopulation,
+					district_id: districtId,
+					facility_id: values[0].id,
+				};
+				const locations = Location.create(locationData);
 
-				dataSource.disconnect();
+				Promise.all([
+					facilityResources,
+					addresses,
+					contacts,
+					facilityServices,
+					facilityUtilities,
+					geolocations,
+					locations,
+				]).then(resvalues => {
+					console.log(
+						"All dependencies in association with\
+					 facilities are created"
+					);
+					dataSource.disconnect();
```

Comment:
make the if statement here:

if (last record) {
    dataSource.disconnect();
}